### ISSUE TITLE
Add Text-to-Speech support to Frag Integreat for improved accessibility

### ISSUE THREAD

**Issue by hauf-toni** [digitalfabrik/integreat-app#3398]
### Describe the Problem

while screen reader users can already access the content we may be excluding users who have mild reading difficulties/low literacy/temporary impairments/limited german skills but do not use assistive technologies. the inconsistency that the tts functionality is not available here could lead to confusion. additionally this creates a discrepancy in the overall accessibility experience across the platform.

### Describe Your Preferred Solution

add optional tts support to Frag Integreat. 

ideally users should be able to trigger tts for individual messages rather than reading the entire chat at once.

### Describe Possible Alternatives

Do not add custom tts + rely solely on screen readers.

### Related Issues

/

### Additional Information

see [original chat](https://chat.tuerantuer.org/digitalfabrik/pl/xwfsyzthp7ds9xgjipczedgb7e) on mattermost.

**Comment by steffenkleinle** [comment]
Perhaps it makes sense to implement this on a per-message basis, i.e. show some options/speaker icon on every message?
It would at least save us work since we wouldn't need to implement this again on native where we use only a web-view to wrap the web chat implementation.
